test/Engine/FFITarget/CodeCompletionClass.cs

```diff
@@ -125,4 +126,16 @@ public class AnotherClassWithNameConflict
             public static string PropertyF { get; set; }
         }
     }
+
+    [IsVisibleInDynamoLibrary(false)]
+    [Serializable]
+    public class TraceableId : ISerializable
```

Comment:
@aparajit-pratap Can you explain how this test works, I can't see how adding a class with the tag verify stuff from my knowledge.

Reply:
@QilongTang I need to think how I can test for interfaces not appearing in search and library.

Reply:
@aparajit-pratap What do you think of either screen comparison or search results comparison?